airbyte-webapp/src/locales/en.json

```diff
@@ -34,11 +34,12 @@
   "form.noConnectorFound": "No matching connector found",
   "form.sourceName.placeholder": "Your source name",
   "form.destinationName.placeholder": "Your destination name",
+  "form.connectionName": "Name",
   "form.connectionName.placeholder": "Name",
   "form.name": "Name *",
   "form.sourceName.message": "Pick a name to help you identify this source in Airbyte",
   "form.destinationName.message": "Pick a name to help you identify this destination in Airbyte",
-  "form.connectionName.message": "Pick a name to help you identify this source in Airbyte",
+  "form.connectionName.message": "Pick a name to help you identify this connection in Airbyte",
```

Comment:
I would remove "in Airbyte" from here because that way the message only appears in one line instead of having "Airbyte" roll into the next line: 
![image](https://user-images.githubusercontent.com/168664/168075048-c3724740-0e62-4096-9a16-2425e940eae5.png)